Labyrinth/Assets/Scripts/TextureGenerator.cs

```diff
@@ -1,6 +1,8 @@
 ﻿/*
  * TextureGenerator.cs
- * Copyright (c) 
+ * Copyright (c) 2017 Rudra Nil Basu <rudra.nil.basu.1996@gmail.com>
+  * AUTHORS
+ * Rudra Nil Basu <rudra.nil.basu.1996@gmail.com>
```

Comment:
Can you maintain the indentation for this line?
Also add your name too, since you are editing the file

Reply:
ok.Doing